packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard-common.spec.tsx

```diff
@@ -88,6 +89,7 @@ describe('Patient dashboard for anyone', () => {
     checkTooltip(deviceUsageWidget, 'Sensor Usage')
 
     expect(patientStatistics.getByTestId('cbg-percentage-bar-chart')).toHaveTextContent('Time In Range2h8%10h42%6h25%4h17%2h8%<5454-7070-180180-250>250mg/dL')
+    expect(patientStatistics.getByTestId('cbg-standard-deviation-stat')).toHaveTextContent('Standard Deviation (61-209)mg/dL74')
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏You could call here the `checkStandardDeviationStatWidget` method. In case we change the data-testid no need to change the code twice ;)